frontend/src/components/RunResultsView.vue

```diff
@@ -0,0 +1,76 @@
+<template>
+  <div>
+    <q-table
+      :rows="runResults"
+      :columns="columns"
+      row-key="run_id"
+      hide-pagination
+      :rows-per-page-options="[0]"
+    />
+  </div>
+</template>
+
+<script lang="ts">
+import { defineComponent } from 'vue';
+export default defineComponent({
+  name: 'RunResultsView',
+});
+</script>
+<script setup lang="ts">
+import { useI18n } from 'vue-i18n';
+import { withDefaults, onMounted, ref, watch } from 'vue';
+import { useRunResultsStore } from 'src/stores/run_results';
+import { Experiment } from 'src/components/models';
+
+const { t } = useI18n({ useScope: 'global' });
+
+export interface RunResultsViewProps {
+  experiment: Experiment;
+}
+const props = withDefaults(defineProps<RunResultsViewProps>(), {
+  experiment: undefined,
+});
+
+const runResultsStore = useRunResultsStore();
+const runResults = ref();
+
+const columns = [
+  'run_id',
```

Comment:
maybe add a Result class in you model file ? So you don't have to list the columns in a component ?

Reply:
There is a RunResult interface in models.ts (though it is incomplete). But how would you turn it into a list of column names?